packages/tailwindcss/src/utilities.test.ts

```diff
@@ -26032,7 +26042,7 @@ describe('spacing utilities', () => {
 
 describe('custom utilities', () => {
   test('custom static utility', async () => {
-    let { build } = await compile(css`
```

Comment:
These format changes seem weird

Reply:
Did Prettier update and change this or something?

Reply:
I was too quick to quit vim and prettier didn't run in time 😂